templates/bin_setup.erb

```diff
@@ -8,7 +8,7 @@ set -e
 
 # Set up Ruby dependencies via Bundler
 gem install bundler --conservative
-bundle check || bundle install
+bundle check || bundle install --path=vendor/bundle
```

Comment:
> IIRC, Travis has configured bundler to install to vendor/bundle. I think the --path argument can be omitted

That only works if you don't use a custom `install` config for Travis http://docs.travis-ci.com/user/caching/#With-a-custom-install-step